src/Fractionalizer.sol

```diff
@@ -147,20 +119,20 @@ contract Fractionalizer is UUPSUpgradeable, OwnableUpgradeable, ReentrancyGuardU
         fractionId = uint256(keccak256(abi.encodePacked(_msgSender(), ipnftId)));
 
         // ensure we can only call this once per sales cycle
-        if (address(fractionalized[fractionId].originalOwner) != address(0)) {
+        if (address(fractionalized[fractionId].tokenContract) != address(0)) {
```

Comment:
What if someone sells the ipnft and buys it again, they won't be able to fractionalize it for the second time

Reply:
Not using the same account, exactly. That’s also lined out on the audit doc and in the real world it’s a highly unlikely usecase :)